app/views/content/explore-teaching-advisers/_article.html.erb

```diff
@@ -1,5 +1,5 @@
-<article>
-  <section class="statement">
+<div class="row inset">
```

Comment:
I expect these classes such as `row`, `inset`, `col` and `col-720` etc will eventually be added to css ?

Reply:
They're already in the layout CSS (may have been added in one of the later commits) https://github.com/DFE-Digital/get-into-teaching-app/pull/2967/files#diff-2fc2fec0b84bdf7c923b7bbf3015aee64123c49bb9dc0518c830a6dbf6ec65f0R170-R180